src/components/RichRadio/RichRadio.tsx

```diff
@@ -91,28 +91,36 @@ type RichRadioItemType = {
 
 export type RichRadioProps = {
   onChange: (newVal: string) => void;
+  renderTextWrapper?: (option: RichRadioItemType) => ReactNode;
   items: RichRadioItemType[];
   value: string;
+  className?: string;
   icon?: ReactNode;
   activeIcon?: ReactNode;
 };
 
 export const RichRadio = ({
   onChange,
+  renderTextWrapper,
   items,
   value,
   icon,
+  className,
   activeIcon,
 }: RichRadioProps) => {
   return (
-    <StyledRoot onValueChange={onChange} value={value}>
+    <StyledRoot className={className} onValueChange={onChange} value={value}>
       <VStack spacing="space-half">
         {items.map(item => (
           <StyledItem value={item.value} key={item.value}>
-            <div>
-              <Label>{item.label}</Label>
-              {item.subtitle && <Subtitle>{item.subtitle}</Subtitle>}
-            </div>
+            {renderTextWrapper ? (
+              renderTextWrapper(item)
+            ) : (
```

Comment:
What do you think of `renderCustomLabel` or `renderCustomText`?
We really are doing more than just rendering a wrapper. 🤔

Reply:
That's true! Thanks 😉